python/dgl/dataloading/pytorch/__init__.py

```diff
@@ -427,7 +427,7 @@ class GraphDataLoader:
 
     Parameters
     ----------
-    collate : Function, default is None
+    collate_fn : Function, default is None
```

Comment:
You probably don't need this argument explicitly; `DataLoader` also has an argument called `collate_fn`.

Reply:
I put it here in order to have an easier control over `self.collate`.